modules/pafRtdProvider.md

```diff
@@ -0,0 +1,93 @@
+## Prebid Addressability Framework Real-time Data Submodule
+
+The PAF real-time data module in Prebid has been built so that publishers
+can quickly and easily setup the Prebid Addressability Framework and utilize OneKey.
+This module is used along with the pafIdSysytem to pass PAF data to your partners.
+Both modules are required. This module will pass transmission requests to your partners
+while the pafIdSystem will pass the pafData.
```

Comment:
Is it important to mention that the publisher and the owner of the adapter must have a mutual agreement on the model terms so that they can use this RTD module? Or is there a general rule for the adapters?

Linked discussion: Does the publisher can configure PrebidJs to activate the module for specific adapters and not for others?

Reply:
yes i would like some help with these descriptions, i am not sure what the final model terms are and what we need to say.

The publisher can explicitly configure prebid.js to activate userId modules for specific adapters, but not for RTD modules, which is why this module allows that functionality through params.bidders.

Reply:
re model terms: I wouldn't refer to them as of yet, but yes the idea is that one should only send the PAF transmission to the relevant bidders.

I would suggest to add "Unless all your partners are PAF partners, you should whitelist the ones who are using param.bidders". (Either here, and/or in the bidders param description)

(Maybe crazy) thought: could the RTD module check the id module configuration to autoconfigure instead of taking a parameter ? Or is that level of introspection not possible or authorized?